external/concealer/include/concealer/autoware_universe.hpp

```diff
@@ -59,6 +63,9 @@ class AutowareUniverse : public Autoware
 
   std::atomic<bool> is_stop_requested = false;
 
+  std::atomic<uint8_t> current_control_mode =
+    autoware_auto_vehicle_msgs::msg::ControlModeReport::AUTONOMOUS;
```

Comment:
```suggestion
  std::atomic<std::uint8_t> current_control_mode =
    autoware_auto_vehicle_msgs::msg::ControlModeReport::AUTONOMOUS;
```

Reply:
Fixed in [`d47b6d7` (#1280)](https://github.com/tier4/scenario_simulator_v2/pull/1280/commits/d47b6d7f8f803d4d56ba2c6c57f3fe733b7e1c0a)